tests/x11/network/yast2_network_use_nm.pm

```diff
@@ -11,9 +11,9 @@
 # Maintainer: Nick Singer <nsinger@suse.de>
 # Tags: poo#20306
 
-use base 'x11test';
-use y2x11test 'launch_yast2_module_x11';
-use y2logsstep;
+use base 'y2_guitest';
+use y2_guitest 'launch_yast2_module_x11';
```

Comment:
y2_guitest is used as parent and also imported again as library. I guess we should avoid this.

Reply:
fixed